examples/SharedMemory/PhysicsServerCommandProcessor.cpp

```diff
@@ -2846,6 +2870,11 @@ void PhysicsServerCommandProcessor::deleteDynamicsWorld()
 	delete m_data->m_remoteDebugDrawer;
 	m_data->m_remoteDebugDrawer = 0;
 
+#ifndef SKIP_DEFORMABLE_BODY
```

Comment:
please use
#if !defined (SKIP_DEFORMABLE_BODY) && !defined(SKIP_SOFT_BODY_MULTI_BODY_DYNAMICS_WORLD)
(Test_PhysicsServerDirect in premake build fails, since it only defines SKIP_SOFT_BODY_MULTI_BODY_DYNAMICS_WORLD)

Reply:
Fixed.